openapi2beans/JavaChecker/src/main/java/dev/galasa/openapi2beans/example/generated/BeanToTestArraysWithVariousPrimitiveTypes.java

```diff
@@ -0,0 +1,43 @@
+package dev.galasa.openapi2beans.example.generated;
+
+// A bean to test array property can contain any primitive property type
+public class BeanToTestArraysWithVariousPrimitiveTypes {
+    // Class Variables //
+    private boolean[] aBooleanArray;
+    private int[] anIntArray;
+    private double[] aNumberArray;
+    private String[] aStringArray;
+
+    // Constants //
+
+    public BeanToTestArraysWithVariousPrimitiveTypes () {
+    }
+
+    // Getters //
+    public boolean[] GetABooleanArray() {
+        return this.aBooleanArray;
+    }
+    public int[] GetAnIntArray() {
+        return this.anIntArray;
+    }
+    public double[] GetANumberArray() {
+        return this.aNumberArray;
+    }
+    public String[] GetAStringArray() {
+        return this.aStringArray;
+    }
+
+    // Setters //
+    public void SetABooleanArray(boolean[] aBooleanArray) {
```

Comment:
This is Java code, so naming conventions for methods start with lower-case letters. `GetAStringArray()` should be `getAStringArray()`. Same for `set` methods.